src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/SecurityFactoryInterface.php

```diff
@@ -31,7 +46,18 @@ public function create(ContainerBuilder $container, $id, $config, $userProvider,
      */
     public function getPosition();
 
+    /**
+     * Defines the configuration key used to reference the provider
+     * in the firewall configuration.
+     * 
+     * @return string
```

Comment:
@iltar this is not the key or context of a firewall, it's just the name of the node in the configuration: https://github.com/symfony/security-bundle/blob/7f04a9ae1ff1dd4eeb6ad20b5df48fdd9c2230e3/DependencyInjection/MainConfiguration.php#L285